addons/l10n_sk/data/template/account.tax-sk.csv

```diff
@@ -1,67 +1,185 @@
-"","","","","","","","","","tax","invoice","","","","","","","",""
-"","","","","","","","","","base","refund","","","","","","","",""
-"","","","","","","","","","tax","refund","","","","","","","",""
-"vy_eu_m","0% EU M","EU Goods","0%","0.0","percent","sale","","tax_group_vat_0","base","invoice","","","Dodanie do EU","fiscal_position_template_2","vy_tuz_23,vy_tuz_19,vy_tuz_0","Exempt intra-community supplies - Article 138 of the Directive 2006/112/EC","consu","Oslobodené dodávky v rámci Spoločenstva - článok 138 smernice 2006/112/ES"
-"","","","","","","","","","tax","invoice","","","","","","","",""
-"","","","","","","","","","base","refund","","","","","","","",""
-"","","","","","","","","","tax","refund","","","","","","","",""
-"vy_eu_t","0% EU T","EU Triangular","0%","0.0","percent","sale","","tax_group_vat_0","base","invoice","","","Trojstranné transakcie","fiscal_position_template_2","","Exempt triangular transaction - Article 141 of the Directive 2006/112/EC","","Oslobodené trojstranné transakcie - článok 141 smernice 2006/112/ES"
-"","","","","","","","","","tax","invoice","","","","","","","",""
-"","","","","","","","","","base","refund","","","","","","","",""
-"","","","","","","","","","tax","refund","","","","","","","",""
-"vs_nad_eu","23% EU","EU import","23%","23.0","percent","purchase","","tax_group_vat_23","base","invoice","","","Nadobudnutie z EU","fiscal_position_template_2","vs_tuz_23,vs_tuz_19","","",""
-"","","","","","","","","","tax","invoice","chart_sk_343120","","","","","","",""
-"","","","","","","","","","tax","invoice","chart_sk_343220","-100","","","","","",""
-"","","","","","","","","","base","refund","","","","","","","",""
-"","","","","","","","","","tax","refund","chart_sk_343120","","","","","","",""
-"","","","","","","","","","tax","refund","chart_sk_343220","-100","","","","","",""
-"vy_mimo_eu","0% EXP","Export Outside EU","0%","0.0","percent","sale","","tax_group_vat_0","base","invoice","","","vývoz mimo EU","fiscal_position_template_3","vy_tuz_23,vy_tuz_23_s,vy_tuz_19,vy_tuz_19_s,vy_tuz_0,vy_tuz_0_s","","",""
-"","","","","","","","","","tax","invoice","","","","","","","",""
-"","","","","","","","","","base","refund","","","","","","","",""
-"","","","","","","","","","tax","refund","","","","","","","",""
+"id","name","description","invoice_label","amount","amount_type","type_tax_use","sequence","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@sk","fiscal_position_ids","original_tax_ids","repartition_line_ids/tag_ids","invoice_legal_notes","tax_scope","invoice_legal_notes@sk"
+"vy_tuz_23","23%","VAT sales 23%","23%","23.0","percent","sale","0","tax_group_vat_23","base","invoice","","","DPH na výstupe 23%","fiscal_position_template_1,fp_intra_private","","03","","",""
```

Comment:
There is a column for the sequence, there is a lot of duplicate of sequence, wondering if we should not remove it or correct the sequence